docs/source/en/chat_templating_multimodal.md

```diff
@@ -175,7 +175,7 @@ processed_chat = processor.apply_chat_template(
     add_generation_prompt=True,
     tokenize=True,
     return_dict=True,
-    video_fps=32,
+    video_fps=16,
```

Comment:
video_fps=32 will cause error: `ValueError: When loading the video with fps=32, we computed num_frames=320 which exceeds total_num_frames=300. Check fps or video meta
data.`